pcdsdevices/valve.py

```diff
@@ -248,6 +248,14 @@ class VGC(VRC):
                                       doc='Downstream vacuum device used for'
                                       'interlocking this valve')
 
+class VGC_2S(VGC):
+    """Class for Controlled Gate Valves with 2 setpoints."""
+    at_vac_setpoint_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_SP_DS', kind='config',
+                          doc='AT VAC Set point value for the downstream gauge')
+    setpoint_hysterisis_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_HYS_DS', kind='config',
+                              doc='AT VAC Hysterisis for the downstream setpoint')
+    
+    
```

Comment:
Suggested fixes for the Travis style issues, hope I hit all of them
```suggestion

class VGC_2S(VGC):
    """Class for Controlled Gate Valves with 2 setpoints."""
    at_vac_setpoint_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_SP_DS',
                             kind='config',
                             doc='AT VAC Set point value '
                                 'for the downstream gauge')
    setpoint_hysterisis_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_HYS_DS',
                                 kind='config',
                                 doc='AT VAC Hysterisis for '
                                     'the downstream setpoint')

```

Reply:
I am not following, the changes in the doc? why?

Reply:
pep8 style dictates line lengths < 80 and the arguments in-line with the previous line inside the parentheses. Here's the output from the travis:
```
pcdsdevices/valve.py:251:1: E302 expected 2 blank lines, found 1
pcdsdevices/valve.py:253:80: E501 line too long (80 > 79 characters)
pcdsdevices/valve.py:254:27: E128 continuation line under-indented for visual indent
pcdsdevices/valve.py:254:80: E501 line too long (80 > 79 characters)
pcdsdevices/valve.py:255:80: E501 line too long (85 > 79 characters)
pcdsdevices/valve.py:256:31: E128 continuation line under-indented for visual indent
pcdsdevices/valve.py:256:80: E501 line too long (82 > 79 characters)
pcdsdevices/valve.py:257:1: W293 blank line contains whitespace
pcdsdevices/valve.py:258:1: W293 blank line contains whitespace
pcdsdevices/valve.py:260:1: E303 too many blank lines (3)
```

Reply:
I did not know that. I Thanks!